Tests/RxCocoaTests/KVOObservableTests.swift

```diff
@@ -16,14 +16,17 @@ import RxCocoa
     import Cocoa
 #endif
 
-final class KVOObservableTests : RxTest {
+final class KVOObservableStringBasedKVOTests : RxTest {
```

Comment:
I'll take a closer look at this file tomorrow, don't have the necessary concentration to go through it today.

Reply:
Thank you very much for reviewing my PR 🙇